web-app/django-startup.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+python manage.py makemigrations
+python manage.py migrate
```

Comment:
my understanding is that it's best practice to include migrations in the git history. Are we confident there will never be a situation where a developer wants to start the server without making new migrations and/or migrating?

Reply:
good point!